deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -1395,15 +1397,15 @@ def allreduce_bucket(self, bucket, rank=None, log=None):
 
         tensor_to_allreduce = tensor
 
-        if pg_correctness_test:
+        if pg_correctness_test or self.sequence_parallel_size > 1:
```

Comment:
`self.sequence_parallel_size >1` is now redundant given the ds_config flag, right?